nbodykit/cosmology/background.py

```diff
@@ -137,7 +137,8 @@ def _solve(self):
             raise ValueError('Neither matter or radiation dominated initial condition. Om(a_i) = %g.' % Om
                            + 'This shall not happen for a reasonable cosmology.')
 
-        y = odeint(self.ode, y0, self.lna, atol=0)
+        # solve with critical point at a=1.0, lna=0.
+        y = odeint(self.ode, y0, self.lna, tcrit=[0.], atol=0)
```

Comment:
How does this work exactly? later on we use self.lna so tcrit is not really computed in the results? Then the normalization at z=0 will be wrong.

Reply:
I was going to ping you about this, since I am unfamiliar with this code... ``tcrit`` specifies "critical points" where the integration "takes special care" -- I found that without it, the integration steps to negative z (which we could get around using a_max>1), but that issue is avoided by setting lna=0 as a critical point.

Reply:
I see. The special care probably means not to jump beyond it during the integration in a poorly defined manner.